docs/Features/XSDFeatures.md

```diff
@@ -0,0 +1,29 @@
+# XSD Features
+
+## XSD Tag Auto Close
+
```

Comment:
Im sorry i have not explained correctly. Dont duplicata those features from xml since xsd is an extension of xml. In this page you must show only xsd features like codelens highlight xsd validation

Reply:
So XSD autoclose shouldnt be included then?

Reply:
No please